example/tests/elementActions.test.js

```diff
@@ -1,12 +1,20 @@
 import { Element } from "test-juggler";
+const fs = require("fs").promises;
+const fsExtra = require("fs-extra");
 
 describe("Element Actions", () => {
     const sliceToClick = new Element("[seriesName='seriesx2'] path");
+    const localPath = process.cwd().replace(/\\/g, "/");
```

Comment:
Why replace is needed here, but not in Element.js?

Reply:
In both places, actually.